de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
I thought `list` is expanded and shared with individual users. It makes more sense for `Teams` to behave this way (unshare items when the `team` is deleted or unshare with a user when that user is removed from the `team`. We should probably get some input from science team.